packages/devextreme/testing/tests/DevExpress.data/queryArray.tests.js

```diff
@@ -350,6 +350,20 @@ QUnit.test('filter with functional getter', function(assert) {
     });
 });
 
+QUnit.test('filter with collatorOptions.sensitivity set to "case"', function(assert) {
```

Comment:
Let's add the test case when the ```langParams``` isn't set

But let's clarify the questions about ```langParams``` type (the comment in the ```d.ts``` above) first
Because we would add more test cases to check all possible scenarios :)